### ISSUE TITLE
problem adding form into modal-body

### ISSUE THREAD

**Issue by matteoventuri7** [shaack/bootstrap-show-modal#3]
If i add html form into modal body, my form disappear leaving its inside html.
Example:
let html = '<form method="post">.....</form>';
$.showModal({ title: "my title", body: html });

**Comment by shaack** [comment]
HTML works in the demo page: https://shaack.com/projekte/bootstrap-show-modal/ (see "Complex Form" example). Can you please post a link to an online-example of the failure.

**Comment by matteoventuri7** [comment]
Try this:

let myhtmlform = '<form method="post" action="/my/test">
<input type="text"/>
<input type="submit" />
</form>';

$.showModal({title: "Hello World!", body:myhtmlform })

You will see that my form will disappear.


Il giorno mer 21 nov 2018, 22:44 Stefan Haack <notifications@github.com> ha
scritto:

**Comment by shaack** [comment]
Ah, sorry, I understand. For a quick fix remove the lines 46 und 49 in `bootstrap-show-modal.js`... and I will think about a long term solution, because without these lines my examples will not work with buttons in the modal-footer.
```
'<form>' + // remove this
'<div class="modal-body"></div>' +
'<div class="modal-footer"></div>' +
'</form>' + // and this
```

**Comment by matteoventuri7** [comment]
Yes, i have made same temporary fix ;)
Thank you.


Il giorno gio 22 nov 2018 alle ore 00:01 Stefan Haack <
notifications@github.com> ha scritto:

**Comment by shaack** [comment]
Fixed with 1.1.7 (https://github.com/shaack/bootstrap-show-modal/commit/27b698252128c6d4bdc37741cddb4e49328188f1), now examples work without the `<form>` wrap. Thanks for reporting!
